tasking/serializers/location.py

```diff
@@ -16,11 +17,25 @@ class LocationSerializer(serializers.ModelSerializer):
     Location serializer class
     """
 
+    def to_internal_value(self, data):
```

Comment:
This should not be in the serializer in my view.  Needs to be a custom field, like so:

```python
class GeoPointField(serializers.Field):
    """
    Custom field for GeoPoint
    """

    def to_internal_value(self, data):
        """
        docstring goes here
        """
        geopoint = data.get('geopoint')

        if geopoint is None:
            self.fail('some error message')

        geopoint_split = geopoint.split(',')
        lon = int(geopoint_split[0])
        lat = int(geopoint_split[1])

        return Point(lon, lat)
```

Reply:
Please have a look at: http://www.django-rest-framework.org/api-guide/fields/#examples

Reply:
Also try to not use `long` as a var name.  In certain versions of Python it is a reserved work